databases/core.py

```diff
@@ -43,6 +43,7 @@
 class Database:
     SUPPORTED_BACKENDS = {
         "postgresql": "databases.backends.postgres:PostgresBackend",
+        "postgresql+asyncpg": "databases.backends.postgres:PostgresBackend",
```

Comment:
This has the same value as the above line.
I think we should either remove default database drivers or avoid this duplicate.